jdbc/core/src/main/java/org/apache/shardingsphere/driver/data/pipeline/datasource/creator/ShardingSpherePipelineDataSourceCreator.java

```diff
@@ -41,6 +42,8 @@ public DataSource createPipelineDataSource(final Object dataSourceConfig) throws
         enableRangeQueryForInline(shardingRuleConfig);
         rootConfig.setDatabaseName(null);
         rootConfig.setSchemaName(null);
+        // TODO set a large enough value, make sure when a jdbc streaming query parameter is take effect
+        rootConfig.getProps().put(ConfigurationPropertyKey.MAX_CONNECTIONS_SIZE_PER_QUERY.getKey(), 100000);
```

Comment:
Need some test when data source max connection is less than MAX_CONNECTIONS_SIZE_PER_QUERY

Reply:
When the number of database connections is less than the number of slices, there will be problems